app/controllers/candidate_interface/apply_controller.rb

```diff
@@ -0,0 +1,18 @@
+module CandidateInterface
+  class ApplyController < CandidateInterfaceController
+    def show
+      @provider_code = params.fetch(:providerCode)
+      @course_code = params.fetch(:courseCode)
+
+      @course = Course
```

Comment:
In the future it would be worth it to pull this + the rescues into a class, so that the controller is not responsible for handling errors from `JsonApiClient`.

Reply:
I think that pattern is worth implementing now, I agree!

Reply:
Have implemented something in this vein.